docs/FileHandler.puml

```diff
@@ -2,16 +2,16 @@
 'https://plantuml.com/class-diagram
 
 class FileHandler {
-  +initFile(Records): void
```

Comment:
Please add some class level attributes in the class diagram too:
Below is copied from the code:
private static final Logger logger = Logger.getLogger(FileHandler.class.getName());
private static String filePath = "data" + File.separator + "bookbob_data.txt";
private static String appointmentFilePath = "data" + File.separator + "bookbob_appointment.txt";

Reply:
noted